packages/opentelemetry-api/package.json

```diff
@@ -45,6 +45,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
```

Comment:
API now depends on scope base because that is where context is. Context is a separate mechanism outside of the API and the SDK, but which is depended on by both.

Reply:
Maybe `Context` and the dummy implementation should be moved like it was done with a other parts recently?

Reply:
I would be willing to move Context into it's own package. @mayurkale22 wdyt?

Reply:
Still thinking on this, I will also check how other libraries are doing it.

Reply:
When we started it we decided to move it back so it could be used by other projects. However since it's started to get heavily linked to the OT specs (and not storing generic context) i would be in favor to move it

Reply:
@vmarchaud I'm sorry I don't understand, you would be in favor of context having it's own package or you would be in favor of moving it into `core`/`api`/other?